contrib/lang/c-c++/packages.el

```diff
@@ -22,19 +23,34 @@ which require an initialization must be listed explicitly in the list.")
 (defun c-c++/init-cc-mode ()
   (use-package cc-mode
     :defer t
-    :config
+    :init
```

Comment:
Sorry my mistake, this should go under `:config`, we don't want to enable semantic when emacs starts.